Makefile

```diff
@@ -71,4 +71,7 @@ endif
 	mkdir -p _vendor
 	mv vendor _vendor/src
 
+simulator:
```

Comment:
it can't by triggered by make dev, ci will not check it. Should we add it in dev command?

Reply:
yep, PTAL